mdx-web/src/main/java/com/mx/path/model/mdx/web/controller/PaymentsController.java

```diff
@@ -17,6 +19,30 @@
 @RequestMapping(value = "{clientId}/users/{user_id}", produces = BaseController.MDX_MEDIA)
 public class PaymentsController extends BaseController {
 
+  @RequestMapping(value = "/payments/enrollment", method = RequestMethod.GET, consumes = BaseController.MDX_MEDIA)
```

Comment:
GET methods don't need `consumes = BaseController.MDX_MEDIA` since there is no request body to consume.